jest.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
+  env: {
+    jest: true,
+  },
+  extends: ['plugin:jest/recommended', './base.js'],
```

Comment:
It works. If with `../jest.js` as last in the array, the prettier config as well as the jest config is applied. This insinuates that putting ../base last in each of the subconfigs is sufficient and the other prettier rules are redundant

<img width="524" alt="screen shot 2017-07-04 at 00 13 44" src="https://user-images.githubusercontent.com/6270048/27809201-c8574e64-604d-11e7-82e1-608ac9310794.png">